odml/tools/rdf_converter.py

```diff
@@ -49,7 +57,12 @@ def save_element(self, e, node=None):
         else:
             curr_node = node
 
-        self.g.add((curr_node, RDF.type, URIRef(fmt.rdf_type())))
+        if fmt._name == "section":
+            s = self._get_section_subclass(e)
+            self.g.add((curr_node, RDF.type, URIRef(s))) if s \
```

Comment:
Use ternary construct only if it fits in one line, otherwise its hard to read.

Maybe simplify to:
```python
u = s if s else u = fmt.rdf_type()
self.g.add((curr_node, RDF.type, URIRef(u)))
```